consensus/tendermint/tmdb_test.go

```diff
@@ -0,0 +1,159 @@
+package tendermint
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/db/pebble"
+	"github.com/NethermindEth/juno/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Test helper to get a TMDBInterface instance
+func newTestTMDB(t *testing.T) TendermintDB[value, felt.Felt, felt.Felt] {
+	dbPath := t.TempDir()
```

Comment:
I think this file should be shared between multiple tests, so we can write tests to assert that the writes are persistent